internal/generate/packagemanifest/packagemanifest.go

```diff
@@ -154,25 +144,9 @@ func (g *Generator) generate() (*apimanifests.PackageManifest, error) {
 	return base, nil
 }
 
-// makeBaseGetter returns a function that gets a base from inputDir.
-func (g Generator) makeBaseGetter(inputDir string) func() (*apimanifests.PackageManifest, error) {
-	basePath := filepath.Join(inputDir, makePkgManFileName(g.OperatorName))
-	if genutil.IsNotExist(basePath) {
-		basePath = ""
-	}
-
-	return func() (*apimanifests.PackageManifest, error) {
-		b := bases.PackageManifest{
-			PackageName: g.OperatorName,
-			BasePath:    basePath,
-		}
-		return b.GetBase()
-	}
-}
-
 // makePkgManFileName will return the file name of a PackageManifest.
 func makePkgManFileName(operatorName string) string {
-	return operatorName + packageManifestFileExt
+	return strings.ToLower(operatorName) + packageManifestFileExt
```

Comment:
This is a breaking change unfortunately.

Reply:
is it? it's been a while since i wrote that bit but I seem to remember i had to change it to keep up with established behavior in the e2e tests somewhere, let me look around

Reply:
so, it looks like this field is subject to RFC 1123 as it's used for the name of kube objects elsewhere. If you try to run this on master right now with a PROJECT file that somehow has a capital project name or manually set it, operator-sdk blows up. With this change, operator-SDK still blows up in the same place. So this line should never actually do anything in actual use, it's just internal hardening. I suppose silently changing it is a bad idea though. Would checking if this field looks right and returning an error if it doesn't be OK?

Reply:
What do you mean by "blows up"? What if you set `generate packagemanifests --package Foo`?

This filename does not have to follow RFC 1123 because it is a filename. RFC 1123 applies to object names though.

Reply:
on current master:
```
jberkhahn@Purgatory> operator-sdk generate packagemanifests --version 1.0.0 --deploy-dir foo --crds-dir api/v1/ --package FOOBAR
Generating package manifests version 1.0.0
ERRO[0000] ClusterServiceVersion validation: [CSVFileNotValid] (FOOBAR.v1.0.0) metadata.name "FOOBAR.v1.0.0" is invalid:
[a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')] 
FATA[0000] Error generating package manifests: error generating ClusterServiceVersion: invalid generated ClusterServiceVersion 
```

Reply:
Ah then perhaps there's no way for a package manifest file name to contain upper case characters because of preceding checks. In that case, lowercase the operator name.

Reply:
after thinking about this some more, i think the real place to fix this is in the validator in operator-framework/api. removing it from this PR